### ISSUE TITLE
Obsolete 'induction by symbiont of host defense response' & children

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#21260]
From today's multiorganism call - the group proposes to obsolete the following terms: 

- [x] 'induction by symbiont of host defense response'
- [x] 'induction by symbiont of defense-related host calcium ion flux'
- [x] 'induction by symbiont of defense-related host MAP kinase-mediated signal transduction pathway'
- [x] 'induction by symbiont of defense-related host nitric oxide production'
- [x] 'induction by symbiont of host ethylene-mediated defense response'
- [x] 'induction by symbiont of host systemic acquired resistance'
- [x] 'induction by virus of host cytokine production'
- [x] 'induction by virus of host immune response'
- [x] 'induction of cell-mediated immune response in host by virus'
- [x] 'induction of humoral immune response in host by virus'
- [x] 'induction of innate immune response in host by virus'
- [x] 'induction by symbiont of host innate immune response'

The reason for obsoletion is that this is not an evolved process of the symbiont's protein.

**Comment by pgaudet** [comment]
Suggestion for replacement: GO:0140418 effector-mediated modulation of host process by symbiont

**Comment by pgaudet** [comment]
This is the list of 'induction by symbiont of host defense response' & children

ID | Term label | Number of annotations
-- | -- | --
GO:0044416 | induction   by symbiont of host defense response | 27
GO:0052391 | induction by symbiont of defense-related host calcium ion flux | 5
GO:0044832 | induction by virus of host cytokine production | 1
GO:0052079 | induction by symbiont of defense-related host MAP kinase-mediated signal transduction pathway | 0
GO:0052063 | induction by symbiont of defense-related host nitric oxide production | 0
GO:0052072 | induction by symbiont of defense-related host salicylic acid-mediated signal transduction pathway | 0
GO:0052076 | induction by symbiont of host ethylene-mediated defense response | 0
GO:0052104 | induction by symbiont of host systemic acquired resistance | 0
GO:0046730 | induction by virus of host immune response | 0
GO:0046737 | induction of cell-mediated immune response in host by virus | 0
GO:0046736 | induction of humoral immune response in host by virus | 0
GO:0046738 | induction of innate immune response in host by virus | 0
GO:0052102 | positive regulation by symbiont of defense-related host calcium-dependent   protein kinase pathway | 0
GO:0052075 | positive regulation by symbiont of host inflammatory response | 0
GO:0052075 | positive regulation by symbiont of host jasmonic acid-mediated defense response | 0
GO:0052074 | positive regulation by symbiont of host salicylic acid-mediated defense response | 0

**Comment by pgaudet** [comment]
Annotations are here: https://docs.google.com/spreadsheets/d/16tQTJxeeHo80b_-UmL0-jogs9LmzlZS5WtNvr7IV9-c/edit#gid=0

**Comment by genegodbold** [comment]
So I don’t think like how this terms is phrased, “induction of host cytokine production”.
It sounds like it could be just part of the normal innate immune response. Keeping things ambiguous in ontologies is not good.

Here are some examples from bacteria in which the symbiont/parasite sequence appears to be doing more than being a recipient of a host response. In these cases (some are ambiguous), it seems as if the parasite protein is goading the host system to produce more inflammation through direct activity:

1)	Hemolysin co-regulated protein 1 (Hcp1) from E. coli (G8IRM9): After six hours of incubation secreted Hcp1, but not Hcp2, causes significantly higher levels of host HBMEC release of interleukin-1 (IL-1) and IL-8 [PMID22184413].
2)	Alpha-hemolysin from Staphylococcus aureus (P09616) facilitates the generation of CXC chemokine gradients in a posttranscriptional manner and potentiates CXC chemokine-induced homing of neutrophils into the lung and airway. This creates neutrophil-mediated inflammation, which is necessary for bacterial attachment and invasion [PMID18823272]. Alpha-hemolysin activates pyrin domain containing 3 inflammasome (NLRP3) to induce production of interleukin-1beta and pyroptosis [but is this a reaction or an action?]. NLRP-3 induced necrosis causes severe pulmonary injury and pneumonia in its host. Nlrp3-/- mice showed significantly lower mortality than wild-type mice [PMID22279123].
3)	Anthrax lethal factor (P15917; Q52NH3; D8HA97; I3XID8; A0A0F7RN98; A0A089G6U8) activates the host nucleotide-binding domain leucine-rich repeat pyrin domain-containing 1B (NLRP1B) inflammasome by cleaving NLRP1B after Lys44 (PMID30872531]. So, this looks like an actual activity—anthrax lethal factor is pushing this process along in the host.
4)	Borrelia burgdorferi B07 (O50983) has RGD motifs and associates with alpha3beta1 integrins during a mouse model of infection [PMID17822440]. This association activates matrix metalloproteases and proinflammatory cytokines that is mediated by JNK but not p38 MAPK [PMID16785564].
5)	Binding of Toxic Shock Syndrome toxin-1 from S. aureus  to host epithelial cells results in [?] the production of host IL-8, MIP-3alpha, and TNF, leading to a proinflammatory environment. The secreted cytokines act to recruit adaptive immune cells to the submucosa where TSST-1 can act upon them [PMID21535475].
6)	Both TcdA and TcdB from Clostridioides difficile are able to activate the inflammasome to secrete interleukin-1beta with TcdB able to induce inflammasome activation at much lower concentrations than TcdA [PMID20398664]. TcdA induction of the inflammasome in infected cells  is dependent on its ability to glucosylate the host Rho protein. This glucosylation is absolutely required for tissue damage and an inflammatory response in vivo [PMID27271747]. Toxin A has a median lethal dose in mice of 4.5 micrograms per kg of bodyweight (90 ng per mouse) [PMID21199912]. It can cause disease in the absence of TcdB [PMID21615333].
7)	C. jejuni cytolethal distending toxin (CDT) directly mediates the release of proinflammatory IL-8 from intestinal epithelial cells [PMID19307212].
8)	CagA from Helicobacter pylori can induce IL-8 production in a time-dependent and strain-dependent manner [PMID20938460]. CagA associates with the host TGFbeta-activated kinase 1 (TAK1) at the plasma membrane, enhancing its activity. Polyubiquitination of TAK1 is important for the activation of TAK1 (and IKK). This requires TRAF6 [PMID21326919]. CagA is required for H. pylori-mediated activation of nuclear factor kappa B [PMID19820695].
9)	Clostridioides difficile transferase (CDT—there are A and B subunits) induces pathogenic host inflammation through a TLR2-dependent pathway which suppresses the protective host eosinophilic response. Bacterial mutants lacking CdtB induced far less IL-1beta and IL-6 in mouse cecal tissue than wild-type bacteria. CDT alone, in the absence of the other C. difficile toxins, can induce NFkappaB activation with expression of IL-1beta and this was blocked with anti-CDT nanobodies. TLR2-deficient mice were protected against the CDT-producing bacterial strain, correlating with increased colonic eosinophils compared to wild type mice [PMID27573114]. CDT recognizes the TLR2/TLR6 heterodimer to induce the NFkappaB response [PMID33011801]. The C. difficile bacterium can be pathogenic expressing only CDT but not toxins A (TcdA) and B (TcdB) [PMID19723585].
10)	The cell translocating kinase A (CtkA; Q9ZKJ5) from H. pylori induces production of pro-inflammatory cytokines (TNF, IL-6, IL-1 beta) in gastric epithelial cells via activation of the host inflammasome [PMID25172221]. CtkA produced by Helicobacter pylori and secreted by a specialized T4SS mediates pro-inflammatory signaling through NFkappaB in host gastric epithelial cells [PMID28759055]. Recombinant CtkA that is catalytically active produces enhanced phosphorylation of the NFkappaB p65 subunit at Ser276 in human epithelial cancer cells [PMID21098302].
11)	ESX secretion-associated protein C (EspC; Rv3615c; P9WJD7) from Mycobacterium tuberculosis expressed in M. smegmatis interacts directly with host TLR4 and greatly affects macrophage activation, stimulating production of host cytokines tumor necrosis factor (TNF), IL-6 and MCP-1 via MAP kinase pathways to activate the innate immune response in a generally pro-inflammatory manner [PMID31134163].
12)	ExoU appears to contribute to host damage by P. aeruginosa (O34208) by inducing an eicosanoid-mediated inflammatory response in the host [PMID16309466]. In Pseudomonas aeruginosa-infected human epithelial cell lines and endothelial cell lines, ExoU phospholipase A2 activity induces NFkappaB and subsequently increases IL-8 levels. These results are also observed in a murine model of P. aeruginosa pneumonia [PMID22848596].

**Comment by pgaudet** [comment]
GO:0052391 induction by symbiont of defense-related host calcium ion flux
GO:0052075 positive regulation by symbiont of host jasmonic acid-mediated defense response
GO:0052104 induction by symbiont of host systemic acquired resistance
GO:0044832 induction by virus of host cytokine production
GO:0052102 positive regulation by symbiont of defense-related host calcium-dependent protein kinase pathway'
GO:0052074 positive regulation by symbiont of host salicylic acid-mediated defense response 
GO:0052035 positive regulation by symbiont of host inflammatory response

**Comment by genegodbold** [comment]
I'm up for this when we are all back from vacation.

**Comment by pgaudet** [comment]
- [ ] GO:0052064 induction by symbiont of defense-related host reactive oxygen species production
keep but change label to 'symbiont-mediated activation of reactive oxygen species production in host cell'
Cite PMID:30823485

**Comment by pgaudet** [comment]
Obsolete GO:0052103 induction of host induced systemic resistance and replace by [induced systemic resistance](http://purl.obolibrary.org/obo/GO_0009682); represents the same concept.

**Comment by genegodbold** [comment]
@pgaudet I don't think induced systemic resistance is the same thing, but I may be misunderstanding this. Is the parasite the agent in these cases or the host? In the examples I gave, there are some where the parasite/symbiont is clearly the agent in activating the existing host machinery to produce (or aggravate) an inflammatory response.

**Comment by pgaudet** [comment]
so, you think there is a difference between 'induction of host induced systemic resistance' and 'induced systemic resistance ' ? The definitions sound the same: 
-   GO:0052103 induction of host induced systemic resistance : Any process in which a symbiont activates, maintains or increases the frequency, rate or extent of induced systemic resistance in the host organism; induced systemic resistance is a response that confers broad spectrum systemic resistance to disease and that does not depend upon salicylic acid signaling. (no references, no anntations)
-   GO:0009682 'induced systemic resistance: A response to non-pathogenic bacteria that confers broad spectrum systemic resistance to disease that does not depend upon salicylic acid signaling.

or do you think was GO:0052103 meant to annotate symbiont genes responsible for GO:0009682?

**Comment by genegodbold** [comment]
A few things:
1) GO:0052103 seems designed for a plant immunity situation, this is not language that mammalian immunologists typically use
2) GO:0009682 is NOT what is occurring in the examples I gave; these are all pathogenic interactions
3) The teleology seems wrong. My thinking is that the symbiont **must** be getting an advantage through this activation of the host inflammatory pathway, otherwise mutations that prevented that activation would have arisen (if they could, that's a point). The way the term is phrased, it seems like the host is benefitting.

**Comment by pgaudet** [comment]
I am wondering if, to replace some of these annotations, a term like 'antigenic activity' could be useful? that would be defined along the lines of ‘A gene product that is recognized by the host cell and that triggers an immune or inflammatory response in the host’.
 
It’s a bit outside the scope of GO but it would provide a way to capture this information. It could go under ‘exogenous protein binding', ie under ‘binding’, so it would not a a ‘real’ MF. 

@jlewsmith @genegodbold @thomaspd @mgiglio99 @ValWood @dsiegele @Antonialock

**Comment by deustp01** [comment]
> ‘A gene product that is recognized by the host cell ...

Restricting "antigen" to mean only the subset of antigens that are encoded gene products is a good fit to the scope of GO - linking gene products to functions - but seems much too restrictive in terms of biology. Entities other than gene products act as antigens, and the immune system does not respond in cleanly distinguishable ways to protein and non-protein antigens. @addiehl , want to jump in here?

**Comment by genegodbold** [comment]
I definitely want Alex's take on this.
We’ve talked about how there should be a way to designate that a microbial protein can trigger the host immune system.

Innate Immune System Triggers: These include lipopolysaccharide for TLR4s, lipoproteins for TLR2s, and also for broad signals like a sudden ion influx or small GTPase hijackings that trigger inflammatory pathways. These are stable signals for the innate immune system of a given organism. It would be do-able to have a term for these different types of triggers. Whether it’s a valuable use of time for most people is another question, but *I* would appreciate having terms like this available to mark microbial sequences that trigger these responses.

Adaptive Immune System Triggers: I think it’s hopeless to try to document these on the provoking/microbial side. Almost anything can be an antigen for the adaptive immune system (B-cell/T-cell). I can’t remember the number of antigens that can be recognized by the adaptive cells of a single organism and I learned about the V/D/J recombinant possibilities in the 1990s, but it was on the order of hundreds of millions to hundreds of billions of possibilities. Almost every sequence could be tagged with "antigenic activity".

**Comment by Antonialock** [comment]
> I am wondering if, to replace some of these annotations, a term like 'antigenic activity' could be useful? that would be defined along the lines of ‘A gene product that is recognized by the host cell and that triggers an immune or inflammatory response in the host’.

Apologies if I'm a bit behind on this discussion. 

If a symbiont gene product has evolved to bind a host receptor to trigger an immune response (which presumably ultimately benefits the pathogen) then it sounds like a receptor ligand? Like an effector receptor ligand? Is that what we are talkinga bout here? (I presume it's not any old thing that inadvertently triggers the immune response)

**Comment by pgaudet** [comment]
I am not aware of a symbiont gene product that has evolved to bind a host receptor to trigger an immune response. 

All symbiont 'patterns' recognized by the host (LPS and other cell wall molecules, DNA, RNA) are being recognized because they are distinctive compare to host molecules. The best the pathogen can do is hide these features or inactivate host receptors.

**Comment by Antonialock** [comment]
if the pathogen protein inactivates the host receptor then it's an inhibitory ligand?

If the pathogen protein is just sensed "normally" by the host then I would only annotate the host protein to MF X + extension: has_input: pathogen protein

**Comment by ValWood** [comment]
>I am not aware of a symbiont gene product that has evolved to bind a host receptor to trigger an immune response.
fungal pathogens do this ( necrotrophs)

**Comment by Antonialock** [comment]
> fungal pathogens do this ( necrotrophs)

I couldn't remember if necrotrophs bind receptors or if they stimulate signaling intracellularly

**Comment by ValWood** [comment]
I think usually they bind intracellular receptors (these receptors are presumably intracelular to detect pathogens which have already invaded?)

This is an example which is an archetype for a fungal plant pathogen
<img width="965" alt="Screenshot 2024-02-01 at 14 47 49" src="https://github.com/geneontology/go-ontology/assets/7359272/43578bb3-5cb5-4e38-bdbf-4b76f7064196">

the plant necrotroph H. arabidopsidis (downy mildew) effector ATR1 binds to the cognate 
plant immune receptor (I think this is intracellular)

**Comment by ValWood** [comment]
This is a hemi-biotroph not a necrotroph, but it has still evolved to activate the host immune system  to turn it on itself.

**Comment by mgiglio99** [comment]
Some bacterial pathogens do this as well.

**Comment by pgaudet** [comment]
@Antonialock 

> if the pathogen protein inactivates the host receptor then it's an inhibitory ligand?

Right, in this case I would annotate to terms such as 'GO:0075109 'symbiont-mediated perturbation of host receptor-mediated signal transduction'
'GO:0052170 symbiont-mediated suppression of host innate immune response' or 'GO:0039537 symbiont-mediated suppression of cytoplasmic pattern recognition receptor signaling pathway' (although that latter should probably not be cytoplasmic-specific; that's on my list to check..) 

Pascale

**Comment by genegodbold** [comment]
Dear @ValWood, isn't this co-evolution? The necrotroph/hemibiotroph is taking advantage of the plant host program that defends itself by destroying the outer layer of infected cells. The host effectors that trigger cell death *often* work to successfully defend the plant.

**Comment by ValWood** [comment]
>But we have more specific terms for this, as the one you showed; or are you saying that we should keep induction by symbiont of host defense response?

sorry yes
GO:0140415 effector-mediated modulation of host defenses by symbiont
or descendants are good. I forgot the overall objective here.

@genegodbold  yes would be co-evolution (arms race, sometimes the plant wins, sometimes the fungus! depending on the strain combination)

**Comment by mgiglio99** [comment]
I think this sub thread was in response to Pascale's suggestion about creating a possible term for 'antigenic activity' and whether this would encompass both "intentional" and "non-intentional" activation of the immune system.

**Comment by addiehl** [comment]
Taken in part from an email I sent earlier today to Gene and Pascale:

'antigenic activity' is really a bad idea for a GO term.  Basically any protein from any source, whether from pathogen or host itself, can have antigenic activity in the right context.  It is not an evolved feature of proteins in general to be antigenic, it is an evolved feature of immune systems to make judgments on whether a particular protein in a particular location warrants a response.  Furthermore, lipopolysaccharide, lipoproteins, etc. did not initially evolve as triggers for the immune system, rather the immune system evolved to recognize them, and then some bacterial species no doubt evolved further to utilize production of these “superantigens” (to use an outdated immunological terminology) as part of their life plan. And of course, anytime I use the word evolve here, I really mean “were favored via natural selection,” to emphasize that evolution is not a conscious decision by a species.  Also, in my mind, antigens are the targets of adaptive immune receptors, such as immunoglobulins or T cell receptors, and not the targets of innate immune receptors such as the TLRs.  In a proper ontological treatment, a protein has the ‘antigen role’ only when it is recognized by an adaptive immune receptor.
 
So if you adopt ‘antigenic activity’, you will be hard pressed to find a definition that excludes basically the full universe of proteins. A GO annotator’s nightmare.
 
In situations where there exists a non-host protein whose structure is favored via natural selection to drive a host immune response, or more correctly modulate an immune response, I am in favor of GO terms such as ‘modulation of host immune response’ (or whatever the current approved language is) and variations thereof.

**Comment by pgaudet** [comment]
@Antonialock the discussion diverged quite a bit. What I wanted to do was to obsolete the two remaining terms above: 
 'induction by symbiont of host defense response'
'induction by symbiont of defense-related host calcium ion flux'
(all others have been obsoleted because they didn't have any annotations or annotations had been removed). 

I proposed to create a MF to capture the genes currently annotated to these terms, but I am happy to not create terms that are out of scope for GO. 

Thanks everyone for the feedback. 

Pascale
